src/transformation/visitors/call.ts

```diff
@@ -211,6 +218,8 @@ function transformElementCall(context: TransformationContext, node: ts.CallExpre
 
 export const transformCallExpression: FunctionVisitor<ts.CallExpression> = (node, context) => {
     const calledExpression = getCalledExpression(node);
+    const symbol = context.checker.getSymbolAtLocation(calledExpression);
+    const customName = getCustomNameFromSymbol(symbol);
```

Comment:
I would put this only in the transformPropertyCall function, as you only pass it there in this function (the transformElementCall seems useless to me, see the other comment). Requesting it this early has a huge performance impact so try to delay it until we are sure we need it.